### ISSUE TITLE
Cydara, Letícia e demais no BookCloud

### ISSUE THREAD

**Issue by hjbortol** [livro-aberto/BookCloud#184]
@simas,

É com você: é preciso (nem que seja ligando e fazendo o procedimento passo a passo junto com a pessoa) incluir Letícia, Cydara e demais nos projetos BookClud e Ensino Médio no GitHub. Se vamos usar este ambiente para discutir e tomar decisões, é preciso que todos estejam cadastrados e associados.

**Comment by simas0** [comment]
Reenviei os convites e novo e-mail solicitando às pessoas que se inscrevam. Se for necessário, semana que vem ligo para cada um deles.

**Comment by hjbortol** [comment]
Acho que só falta a Letícia e o Ney, não é?

**Comment by simas0** [comment]
Sim, Humberto, ainda faltam a Letícia, o Ney e a Vanessa. Já liguei para eles hoje e ambos confirmaram que aceitarão ao convite ainda hoje.

**Comment by estelanunes** [comment]
Estou aqui, Humberto!!!

**Comment by hjbortol** [comment]
:+1::+1::+1:
